### ISSUE TITLE
Wrong IP of web sites if launch FinalRecon through proxychains

### ISSUE THREAD

**Issue by Mi-Al** [thewhiteh4t/FinalRecon#10]
Hello! When I start FinalRecon through proxychains like this:

`sudo proxychains4 finalrecon https://site.biz`

I always get wrong IP address for every web site, it is 224.0.0.2:

`[+] IP Address : 224.0.0.2`

But information from DNS Enumeration is correct anyway.

I tried to specify custom DNS with -d option, but it did not help.

**Comment by thewhiteh4t** [comment]
I have not tested it with proxychains yet, I will try and fix this soon, thanks!

**Comment by thewhiteh4t** [comment]
@Mi-Al I tested with both `proxychains` and `torsocks` , I am using tor in proxychains, IP changes like you mentioned but that's not the case with torsocks, please test using torsocks also

**Comment by Mi-Al** [comment]
Yes, with torsocks FinalRecon works fine. I can use torsocks instead of proxychains. Thank you!

**Comment by thewhiteh4t** [comment]
@Mi-Al Great! thanks, I am testing support for proxychains at the moment
